src/tiler/sql/user_reservable.sql

```diff
@@ -1,24 +1,33 @@
 (SELECT
-  block.geom, block.id, turf.group_id,
-  CASE
-    WHEN (block.is_available AND reservation.id IS NULL
-          AND (turf.id IS NULL OR trustedmapper.id IS NOT NULL)) THEN 'available'
-    ELSE 'unavailable'
-  END AS survey_type,
-  CASE
-    WHEN turf.id IS NOT NULL AND trustedmapper.id IS NULL THEN 'group_territory'
-    WHEN reservation.id IS NOT NULL THEN 'reserved'
-    WHEN NOT block.is_available THEN 'unavailable'
-    ELSE 'none'
-  END AS restriction
-  FROM survey_blockface AS block
-  LEFT OUTER JOIN survey_territory AS turf
-    ON block.id = turf.blockface_id
-  LEFT OUTER JOIN survey_blockfacereservation AS reservation
-    ON (block.id = reservation.blockface_id
-        AND reservation.canceled_at IS NULL
-        AND reservation.expires_at > now() at time zone 'utc')
-  LEFT OUTER JOIN users_trustedmapper AS trustedmapper
-    ON (turf.group_id = trustedmapper.group_id
-        AND trustedmapper.user_id = <%= user_id %>)
+    <% if (is_utf_grid) { %>
+    CASE
+      WHEN survey_type = 'available' THEN ST_AsGeoJSON(geom)
+      ELSE NULL
+    END AS geojson,
```

Comment:
I tried that to begin with.

You can't reference a computed column like `survey_type` from within the same query, which was what prompted me to move the bulk of this SQL statement into a subquery.